script.js

```diff
@@ -1,4 +1,109 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  console.log(`Round is ${roundCounter}`);
+  if (roundCounter == `player1Rolls`) {
+    for (var counter = 0; counter < 2; counter += 1) {
+      var roll = dice[Math.floor(Math.random() * dice.length)];
+      playerRolls.push(roll);
+      console.log(`${roll}`);
+    }
+    roundCounter = `player1Chooses`;
+    var rollText =
+      `Welcome Player 1<br><br>` + rollSentence(playerRolls[0], playerRolls[1]);
+    // doesn't work: return `Welcome Player 1.<br><br>${rollSentence}`;
+    // doesn't work: var rollSentence = rollSentence (playerRolls[0], playerRolls[1])
+    // doesn't work: return `Welcome Player 1.<br><br>` + rollSentence;
+    return rollText;
+  }
+  if (roundCounter == `player1Chooses`) {
+    if (input != 1 && input != 2) {
+      var errorText =
+        rollSentence(playerRolls[0], playerRolls[1]) +
+        `<br><br>Please choose "1" or "2"!`;
+      return errorText;
+    }
+    if (input == 1) {
+      player1Choice = Number(`${playerRolls[0]}${playerRolls[1]}`);
+    }
+    if (input == 2) {
+      player1Choice = Number(`${playerRolls[1]}${playerRolls[0]}`);
+    }
```

Comment:
this can prob just be 

```
player1Choice = Number(`${playerRolls[0]}${playerRolls[1]}`);

if (input == 2) {
      player1Choice = Number(`${playerRolls[1]}${playerRolls[0]}`);
    }
    ```